x-pack/plugins/stack_connectors/server/config.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { PluginInitializerContext } from '@kbn/core-plugins-server';
+import { schema, TypeOf } from '@kbn/config-schema';
+import {
+  getExperimentalAllowedValues,
+  ExperimentalFeatures,
+  isValidExperimentalValue,
+  parseExperimentalConfigValue,
+} from '../common/experimental_features';
+
+const allowedExperimentalValues = getExperimentalAllowedValues();
+
+export type ConfigSchema = TypeOf<typeof configSchema>;
+
+export type ConfigType = ConfigSchema & {
+  experimentalFeatures: ExperimentalFeatures;
+};
+
+export const configSchema = schema.object({
+  enableExperimental: schema.arrayOf(schema.string(), {
+    defaultValue: () => [],
+    validate(list) {
+      for (const key of list) {
+        if (!isValidExperimentalValue(key)) {
+          return `[${key}] is not allowed. Allowed values are: ${allowedExperimentalValues.join(
```

Comment:
We may have a problem with these causing Kibana to fail to start.  We need to make sure we can validate the correct entries in cloud, otherwise if we can't, users could enter invalid values, and Kibana would boot-loop.  I _think_ we can probably do the validation for this in cloud, but _not sure_.

Reply:
Do you think it's a problem that we have this code in [fleet](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/config.ts#L153-L164), [security solution](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/config.ts#L95-L106), and [triggers_actions_ui](https://github.com/elastic/kibana/blob/main/x-pack/plugins/triggers_actions_ui/server/config.ts#L22-L33)?

Reply:
Depends on whether the cloud validations of these catch them before starting the code.  I'll check ...

Reply:
I'm not seeing `triggers_actions_ui` expose this config on cloud, maybe that's another PR.  The other two have a simple validation - array of strings.  So users can enter any strings.  So Kibana will boot-loop if they enter one of those in the cloud.

I'd suggest changing the validation code to not return the error on unknown strings - filter them out and log a warning on unknown strings.

It's possible the cloud validation could be made to check for the exact strings, but the validations are ... _complicated_ ...